kalamine/layout.py

```diff
@@ -203,6 +204,21 @@ def __init__(self, filepath: Path) -> None:
 
         # keyboard layers: self.layers & self.dead_keys
         rows = GEOMETRY[self.meta["geometry"]].rows
+
+        # Adding Angle Mod support
+        if self.is_angle_mod:
+            not_compatible_with_angle_mod = True
+            for row in rows:
```

Comment:
Silly question, can’t you just assume the LSGT key can only be on the 4th row and avoid this loop ?